src/EnliteMonolog/Service/MonologServiceFactory.php

```diff
@@ -57,6 +58,10 @@ public function createLogger($container, MonologOptions $options)
             $logger->pushProcessor($this->createProcessor($container, $processor));
         }
 
+        if (class_exists('\Monolog\ErrorHandler') && $options->isErrorHandler()) {
```

Comment:
:/ Hovewer after this feature I think is good moment to make and release new version with 5.6 as minimum version.